tools/schemas/monorepo-schema.json

```diff
@@ -0,0 +1,60 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "properties": {
+    "$schema": {
+      "type": "string",
+      "oneOf": [
+        { "format": "uri" },
+        { "pattern": "^(\\.\\.?/)*[\\w.-]+(/[\\w.-]+)*\\.json$" }
+      ]
+    },
```

Comment:
do we really need this? 🤔

Reply:
I get a minor wraning from lint when this is not added.

Reply:
use a newer json schema

https://json-schema.org/specification-links#published-drafts

Reply:
We use the same for renovate.json 's schema and it works fine ... 🤔